docs/tables/net_certificate.md

```diff
@@ -41,5 +41,19 @@ from
 where
   domain = 'steampipe.io'
   and not_before < current_timestamp
-  and not_after > current_timestamp
+  and not_after > current_timestamp;
+```
+
+### Check if the certificate was revoked by the CA
```

Comment:
There are a lot of new columns being added into this table, can you please add more examples here showing how to use some of the new columns? The JSON column types in particular would be helpful to show usage for.